tests/FunctionalTestCase.php

```diff
@@ -341,13 +349,12 @@ protected function skipIfChangeStreamIsNotSupported()
 
     protected function skipIfCausalConsistencyIsNotSupported()
     {
-        switch ( $this->getPrimaryServer()->getType() )
-        {
+        switch ( $this->getPrimaryServer()->getType()) {
```

Comment:
I think the space between `(` and `$` can be deleted, since this trimmed the trailing space.

Reply:
I excluded that rule for now since I didn't know whether you want to keep the space before the negation operator. I'll drop the exclude and enforce the rule as per PSR-2.